Library/Formula/zopfli.rb

```diff
@@ -1,21 +1,27 @@
-require 'formula'
-
 class Zopfli < Formula
   desc "New zlib (gzip, deflate) compatible compressor"
-  homepage 'https://code.google.com/p/zopfli/'
-  url 'https://zopfli.googlecode.com/files/zopfli-1.0.0.zip'
-  sha1 '98ea00216e296bf3a13e241d7bc69490042ea7ce'
-  head 'https://code.google.com/p/zopfli/', :using => :git
+  homepage "https://github.com/google/zopfli"
+  url "https://zopfli.googlecode.com/files/zopfli-1.0.0.zip"
+  sha256 "e20d73b56620285e6cce5b510d8e5da6835a81940e48cdf35a69090e666f3adb"
+  head "https://github.com/google/zopfli.git"
 
   def install
-    # Makefile hardcodes gcc
-    inreplace 'makefile', 'gcc', ENV.cc
-    system 'make', '-f', 'makefile'
-    bin.install 'zopfli'
-    if build.head?
-      system 'make', '-f', 'makefile', 'zopflipng'
-      bin.install 'zopflipng'
+    if build.stable?
+      # Makefile hardcodes gcc
+      inreplace "makefile", "gcc", ENV.cc
+      system "make", "-f", "makefile"
+    else
+      inreplace "Makefile", "gcc", ENV.cc
+      inreplace "Makefile", "g++", ENV.cxx
+      inreplace "Makefile", "-soname", "-o"
```

Comment:
If they aren't accepting patches I'm hesitant to include this software in Homebrew as it sounds unmaintained.